src/App.js

```diff
@@ -1,22 +1,60 @@
-import React from 'react';
-import Map from './containers/Map'
+import React, {
+  useState,
+  useEffect
+} from 'react';
+import Map from './containers/Map';
+import Header from './components/Header';
 import './styles/App.css';
 
 
-// Use these co-ordinates to centre the map
-const sanFranciscoCoordinates = {
-      lat: 37.774929,
-      lng: -122.419416
+// Use these co-ordinates to centre the map on San Francisco
+const homeLatLng = {
+  coords: {
+    latitude: 37.774929,
+    longitude: -122.419416
+  }
 }
 
 export default function App() {
 
+  // State: Maps center. This can be set, or loaded from the users' Navigator.
+  // Future: Perhaps calculate optimal center based on current Markers
```

Comment:
Nit:
```suggestion
  // TODO(duckranger): Perhaps calculate optimal center based on current Markers
```
You don't need a name after `TODO`, but The One True Google Way is to put one there.  This is the name of the person to ask about this TODO (not necessarily the person who wrote the TODO or the person who needs to do it).  Or you can put an issue reference there, of course, for extra accountability.

Reply:
Thanks! That's neat :)